invenio_communities/communities/services/service.py

```diff
@@ -714,6 +714,25 @@ def on_relation_update(self, identity, record_type, records_info, notif_time):
 
         return True
 
+    @unit_of_work()
+    def bulk_add_to_parent(self, identity, comm_slugs, new_parent, uow=None):
+        """Bulk adds communities to a new parent.
+
+        :param identity: The identity of the user performing the action.
+        :param comm_slugs: The list of community slugs to add to the new parent.
+        :param new_parent: The slug of the new parent community.
+        """
+        t_comm = self.read(identity, new_parent)
+        for c_slug in comm_slugs:
+            c_comm = self.read(identity, c_slug)
+            self.update(
+                identity,
+                c_slug,
+                data={**c_comm.data, "parent": {"id": t_comm.id}},
```

Comment:
Question: Should there be a check for an already existing parent, or is it okay to replace it? (maybe adding a force option to replace would be useful (?))

To us it looks like if we want to allow updates at least we should call it `bulk_update_parent` instead.

Reply:
Regarding the questions:

1) will be added, I don't see why not. 
2) I also agree with this one. The way I saw it, we needed a way to "transfer" multiple communities to a new parent. Thus, the wording `add to parent`. But you're right, this is updating the parent (not adding).